src/_lib/modules/word-cloud/word.component.html

```diff
@@ -0,0 +1 @@
+{{word.truncatedValue}}
```

Comment:
A lot of these files end in a new line, which git is flagging as a lint error.   Cause for concern?